### DISCUSSION TITLE
How to show a placeholder for empty `NodeView`s?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [ueberdosis/tiptap#3072] [General Questions]
I have a NodeView with an editable section (`<NodeViewContent />`) and I want to show a placeholder if the editable section is empty. How can I do that?

**Comment by sampotts** [comment]
I know this is an old issue but if anyone else runs into this and ends up here via Google, the [Placeholder extension](https://tiptap.dev/api/extensions/placeholder/) has an `includeChildren` option which is `false` by default but needs to be `true` to render inside `NodeViewContent`.

**Reply by Southclaws** [reply]
This doesn't appear to work on the latest version. Is there a workaround you know of? I'm trying to add an alt-text/description input for images.
<img width="243" alt="image" src="https://github.com/ueberdosis/tiptap/assets/1636971/d9ee51dd-a11b-4728-980f-df217748ce3f">


```ts
Placeholder.configure({
        placeholder: "Write your heart out...",
        includeChildren: true,
        showOnlyCurrent: false,
        considerAnyAsEmpty: true,
      }),

...

<NodeViewWrapper className="react-component">
      <styled.img borderRadius="md" {...props.node.attrs} />
      <NodeViewContent
        className={css({
          fontStyle: "italic",
          color: "fg.muted",
        })}
      />
    </NodeViewWrapper>
```
